src/App.tsx

```diff
@@ -17,18 +56,18 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter value={query} selectValue={selectFilter} onSelect={setSelectFilter} onChange={setQuery} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isLoaded && <Loader />}
+              {isLoaded && (<TodoList onSelectedTodo={setTodo} selectedTodo={todo} todos={filterTodos(todos)} onFind={findTodo} />)}
```

Comment:
Linebreaks ? Use ternary operator in this case
```suggestion
              {isLoaded 
               ? (<TodoList 
                     onSelectedTodo={setTodo} 
                     selectedTodo={todo} 
                     todos={filterTodos(todos)} 
                     onFind={findTodo} 
                   />)
              : <Loader />}
```